src/main/java/teammates/ui/webapi/output/SearchSessionData.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.webapi.output;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents a Sessions search result.
+ */
+public class SearchSessionData extends ApiOutput {
```

Comment:
Can we just return a list of `SessionData` here?

Reply:
Hi @xpdavid, in the original response object, each session has two attributes - `sessionName` and `sessionUrl`. Compared to the existing `FeedbackSessionData` object, the only common attribute is the session name. 

In this case, should I: 
- Use the existing `FeedbackSessionData` but remove all the unnecessary fields and add `sessionUrl` or
- Create a new `SearchSessionData` class with only `sessionName` and `sessionUrl`?

Also, we cannot simply return a list of sessions because we need to identify which sessions belong to which user, so that we can join and build the response object on the frontend.